src/snowflake/snowpark/modin/pandas/indexing.py

```diff
@@ -1311,12 +1311,16 @@ def __setitem__(
             set_as_coords = is_row_key_df and is_col_key_df
 
         new_qc = self.qc.set_2d_positional(
-            row_loc._query_compiler
-            if isinstance(row_loc, BasePandasDataset)
-            else row_loc,
-            col_loc._query_compiler
-            if isinstance(col_loc, BasePandasDataset)
-            else col_loc,
+            (
+                row_loc._query_compiler
+                if isinstance(row_loc, BasePandasDataset)
```

Comment:
what is the change we made here? is it just the ()?

Reply:
Yes, it's just a parentheses (it was a linter artifact from when I used `pd.base.BasePandasDataset` instead of importing the name first). I've reverted this.